src/gtest/test_sidechain_ceased.cpp

```diff
@@ -0,0 +1,487 @@
+#include <gtest/gtest.h>
+#include <chainparams.h>
+#include <coins.h>
+#include "tx_creation_utils.h"
+#include <main.h>
+#include <undo.h>
+#include <consensus/validation.h>
+
+class CeasedSidechainsTestSuite: public ::testing::Test {
+
+public:
+    CeasedSidechainsTestSuite():
+        dummyBackingView(nullptr)
+        , view(nullptr) {};
+
+    ~CeasedSidechainsTestSuite() = default;
+
+    void SetUp() override {
+        SelectParams(CBaseChainParams::REGTEST);
+
+        dummyBackingView = new CCoinsView();
+        view = new CCoinsViewCache(dummyBackingView);
+    };
+
+    void TearDown() override {
+        delete view;
+        view = nullptr;
+
+        delete dummyBackingView;
+        dummyBackingView = nullptr;
+    };
+
+protected:
+    CCoinsView        *dummyBackingView;
+    CCoinsViewCache   *view;
+};
+
+
+///////////////////////////////////////////////////////////////////////////////
+/////////////////////////// isSidechainCeased /////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+TEST_F(CeasedSidechainsTestSuite, UnknownSidechainIsNeitherAliveNorCeased) {
+    uint256 scId = uint256S("aaa");
+    int creationHeight = 1912;
+    CTransaction scCreationTx = txCreationUtils::createNewSidechainTxWith(scId, CAmount(10));
```

Comment:
Creating this is probably useless

Reply:
Agree, and it doesn't even help readability. Removed